fileset-plugins-parquet/src/test/java/co/cask/hydrator/plugin/batch/file/ETLSnapshotTestRun.java

```diff
@@ -52,7 +54,8 @@
     Schema.Field.of("id", Schema.of(Schema.Type.STRING)),
     Schema.Field.of("price", Schema.of(Schema.Type.INT)));
 
-  //@Test
+  @Ignore
```

Comment:
@anwar6953 Should the JIRA say that the build failed because the Test annotation was not present and that this PR fixes it?